### DISCUSSION TITLE
TW-Elements in .NET Blazor app

### DISCUSSION THREAD

**Discussion by vsirghii** [mdbootstrap/TW-Elements#2274] [Support from community]
I know that we can use Tailwind in a Blazor app. There are some tutorial in youtube and some blogs.
But I couldn't find anything about TW-Elements. Any links or ideas about how to use Blazor with TW-Elements?
Thank you,

**Comment by iprzybysz** [comment]
Hi, we've worked recently on [ASP.NET integration tutorial](https://github.com/mdbootstrap/TW-Elements/issues/2110) and it will be added soon. It should help you with it.
